venv/task1.py

```diff
@@ -0,0 +1,38 @@
+import connections as con
+
+rb = con.basics.join(con.ratings,
```

Comment:
`rb` what does it mean?  Bad naming
double quotes or single? Please choose one style for whole your project